include/s3select_functions.h

```diff
@@ -956,13 +963,13 @@ struct _fn_like : public base_function
     base_statement* main_expr = *iter;
     value expr_val = expr->eval();
     value main_expr_val = main_expr->eval();
-    std::string a = expr_val.to_string();
-    std::string s = transform(a);
-    std::regex x{s};
+    std::string stringvalue = expr_val.to_string();
+    std::string stringtransform = transform(stringvalue);
+    std::regex x{stringtransform};
```

Comment:
please **note**:
/_std::regex x_/ is defined twice , here , inside /_operator()_/
and also as /__fn_like_/ data member 

it should defined once, and populate once, i.e. transform should run upon construction.